src/code42cli/cmds/departing_employee.py

```diff
@@ -17,12 +17,14 @@
 from code42cli.file_readers import read_csv_arg
 from code42cli.options import format_option
 from code42cli.options import sdk_options
+from code42cli.util import deprecation_warning
 
 
 def _get_filter_choices():
     filters = DepartingEmployeeFilters.choices()
     return get_choices(filters)
 
+DEPRECATION_TEXT = "(DEPRECATED): Use `code42 watchlists` commands instead."
```

Comment:
I had thought about listing out each replacement in each command help, but since they aren't 1-1 in all features didn't want to have to get into the differences. 

I think maybe I'll update the [detections list user-guide](https://clidocs.code42.com/en/stable/userguides/detectionlists.html) and lay out the replacements there, and then add that link to `DEPRECATION_TEXT`. 

This also reminded me I need to add a watchlists user-guide!